pkg/webui/components/notification/index.js

```diff
@@ -65,7 +65,7 @@ const Notification = function({
         <div className={style.content}>
           {title && <Message className={style.title} content={title} component="h4" />}
           <div>
-            <Message content={content} values={messageValues} />
+            <Message content={content} values={messageValues} hasMarkdown={Boolean(error)} />
```

Comment:
```suggestion
            <Message content={content} values={messageValues} convertBackticks={Boolean(error)} />
```

Reply:
Let's call this `convertBackticks`, since we're not really parsing any markdown other than converting backticks.